src/index.js

```diff
@@ -1,9 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter } from 'react-router-dom'
 import App from './components/app';
 import './styles/index.css';
 
 ReactDOM.render(
-  <App url={document.URL}/>,
+  // According to the docs, forceRefresh is required for browsers which don't
+  // support pushState. However, even when this is set to false and 'pushState'
+  // in window.history is 'true', clicked links did not impact the content.
+  <BrowserRouter forceRefresh={true} basename="/qpp-submissions-docs/">
```

Comment:
hm, this is worth digging more into I think. not sure if it's related, but clicking 'back' after navigating off of the root doesn't work either